meson.build

```diff
@@ -0,0 +1,32 @@
+project('multianalyzer',
+  'c', 'cython',
+  license: 'MIT',
+  meson_version: '>= 0.60',
+  version: run_command('version.py',
+  			           check:true).stdout().strip(),
+)
+
+
+# Seek the backend
+if meson.backend() != 'ninja'
+  error('Ninja backend required')
+endif
+
+# Seek for Cython installation
+cython = find_program('cython', required : true)
+if not cython.found()
+  error('Cython not found.')
+endif
```

Comment:
This shouldn't be necessary -- it is already searched for as part of the project() function, since the cython language is listed.

Reply:
Thanks for the hint !